src/Backend/Modules/Location/Engine/Model.php

```diff
@@ -138,11 +138,8 @@ public static function getCoordinates(
         // define address
         $address = implode(' ', $item);
 
-        // define Google Maps API key
-        $apikey = BackendModel::get('fork.settings')->get('Core', 'google_maps_key');
-
         // define url
-        $url = 'http://maps.googleapis.com/maps/api/geocode/json?address=' . rawurlencode($address) . '&key=' . $apikey;
+        $url = 'https://maps.googleapis.com/maps/api/geocode/json?address=' . rawurlencode($address);
```

Comment:
We need two keys if we want to do it right:
- **browser key** to show the google maps correctly
- **server key** to get the geocoordinates (in BackendLocationModel), but this one is optional. That's why I removed it in BackendLocationModel.

Currently we only use the browser key to show the "google maps".